src/components/ui/ProgressBarStep.js

```diff
@@ -0,0 +1,75 @@
+import { Check } from '@styled-icons/boxicons-regular'
+import React from 'react'
+import styled from 'styled-components/macro'
+
+import CircleIcon from '../ui/CircleIcon'
+import { theme } from './GlobalStyle'
+
+const PrimaryText = styled.div`
+  font-size: 15px;
+  color: ${({ color }) => color};
```

Comment:
Make color a transient prop as well

Reply:
Instead of having color as a transient prop, is there a reason we didn't do color: ${({ theme }) => theme.primaryText}?

Reply:
Your color is dynamic, so now you can just switch it based off status. I made this comment before the ones above